samples/subsys/fs/sample.yaml

```diff
@@ -0,0 +1,8 @@
+sample:
+  name: Fat filesystem sample
+tests:
+  test:
+    build_only: true
+    platform_whitelist: nrf52840_blip, olimexino_stm32
```

Comment:
there should be no `,` should be:

`platform_whitelist: nrf52840_blip olimexino_stm32`